samples/subsys/usb/cdc_acm_composite/prj.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_USB_COMPOSITE_DEVICE=y
 CONFIG_USB_CDC_ACM=y
 CONFIG_USB_CDC_ACM_DEVICE_COUNT=2
 CONFIG_USB_CDC_ACM_RINGBUF_SIZE=512
+CONFIG_USB_REQUEST_BUFFER_SIZE=192
```

Comment:
I think its not related to this PR and should be removed or am I missing something?

Reply:
It is based on #19203 and there is no problem with that.

Reply:
Its not a problem with a change itself. This PR is about ZLP handling so I would avoid putting this change here as it is not related.

Reply:
It is related because "Alternatively, cdc_acm_composite sample can be used as it has a string descriptor exact 64 bytes long." To do that #19203 is necessary.